backend/service/audit/storage/sql/sql.go

```diff
@@ -56,9 +54,8 @@ func (c *client) WriteRequestEvent(ctx context.Context, event *auditv1.RequestEv
 		return -1, errors.New("cannot write empty event to table")
 	}
 
-	reqBody, err := convertAPIBody(event.RequestMetadata.Body)
-	if err != nil {
-		return -1, err
+	if !c.filterRequest(event) {
+		return -1, ErrFailedFilters
```

Comment:
This was the only conflict we had. This was the original code https://github.com/lyft/clutch/pull/524/files#diff-037973d02534a9e92ba9793dbb13e9f9c740d4d0af1bbbd3a237422e4a01fbe5R112-R115.